devices/src/virtio/net.rs

```diff
@@ -498,6 +498,13 @@ impl NetEpollHandler {
 }
 
 impl EpollHandler for NetEpollHandler {
+    fn set_queues(&mut self, _other: &Vec<Queue>) {
```

Comment:
perhaps add TODOs here to make sure it's clear these are currently incomplete implementations?